src/test/configuration/index.js

```diff
@@ -3,6 +3,17 @@ const axios = require('axios');
 const { startTestServer, isStageTesting, port } = require('../../server.js');
 axios.defaults.baseURL = `http://localhost:${port}`;
 
+/**
+  * Abstract function for HTTP requests.
+  * @param {string} request.url - The url to make the request to.
+  * @param {string} [request.method] - The method of the request. Defaults to GET.
+  * @param {object} [request.body] - The payload for the request.
+  * @param {object} [request.headers] - The headers to accompany the request.
+  * @param {Function} [request.onSuccess] - Function triggered on successful request.
+  * @param {Function} [request.onError] - Function triggered on successful request.
+  * @param {Function} [request.done] - The callback to finish the test.
+*/
+
```

Comment:
This JSDoc is for the `request` function. Why is it just existing on its own?

Reply:
It was in the module.exports function and I wasnt sure where to put it because I wasnt sure what is was doing. It was part of what I was asking

Reply:
Alright i guess I would put it with the request export tthen

Reply:
I have placed this just about the request export makes sense there

Reply:
Ahh I see. It's used to document what the function does and what each of the arguments are. Ideally, all functions in the codebase should be documented like that, but sometimes laziness did not permit me